scripts/vagrant/Vagrantfile

```diff
@@ -17,7 +23,7 @@ Vagrant.configure("2") do |config|
     v.vmx["memsize"] = "4096"
     v.vmx["numvcpus"] = "2"
   end
-  config.vm.box = "bento/ubuntu-18.04"
+  config.vm.box = "generic/ubuntu1604"
```

Comment:
@matrohon we discussed on the call that it might be a good idea if we make this conditional. I.e. when the provider is libvirt, use `generic/ubuntu1604` otherwise stick with bento 1804. What do you think?

Reply:
done